en/flight_modes/follow_me.md

```diff
@@ -41,45 +49,63 @@ The following flight precautions should be observed:
+
+Therefore using MAVSDK for a Follow Me isn't covered here for now.
 
 ## Configuration
 
-The follow-me behaviour can be configured using the following parameters:
+:::warning
+Do not set the **Altitude mode ([FLW_TGT_ALT_M](#FLW_TGT_ALT_M)**) to `3D Tracking`!
+The MAVLink [FOLLOW_TARGET](https://mavlink.io/en/messages/common.html#FOLLOW_TARGET) message definition requires the Altitude above Mean Sea Level (AMSL) for the target altitude. Testing of QGC on Android phone however showed that it's reporting Altitude above Ellipsoid (which can differ as much as 200 meters!), which is a bug in QGC right now.
+Therefore it's not recommended to turn on 3D tracking mode
+:::
+
+The follow-me behavior can be configured using the following parameters:
 
 Parameter | Description
 --- | ---
-<span id="NAV_FT_DST"></span>[NAV_FT_DST](../advanced_config/parameter_reference.md#NAV_FT_DST) | Vehicle/ground station separation in the *horizontal* (x,y) plane. Minimum allowed separation is 1 meter. Default distance is 8 meters (about 26 ft).
-<span id="NAV_MIN_FT_HT"></span>[NAV_MIN_FT_HT](../advanced_config/parameter_reference.md#NAV_MIN_FT_HT) | Vehicle follow-me height. Note that this height is fixed *relative to the home/arming position* (not the target vehicle). Default and minimum height is 8 meters (about 26 ft).
-<span id="NAV_FT_FS"></span>[NAV_FT_FS](../advanced_config/parameter_reference.md#NAV_FT_FS) | Flight position relative to the user when follow-me mode is active.<br>- `0` = Follow from the front right.<br>- `1` = Follow from behind or trail the user (Default).<br>- `2` = Follow from the front.<br>- `3` = Follow from the front left.
+<a id="FLW_TGT_ALT_M"></a>[FLW_TGT_ALT_M](../advanced_config/parameter_reference.md#FLW_TGT_ALT_M) | Altitude control mode. <br>- `0` = 2D Tracking <br>- `1` = 2D Tracking + Terrain Following <br>- `2` = 3D Tracking of the target's altitude
+<a id="FLW_TGT_HT"></a>[FLW_TGT_HT](../advanced_config/parameter_reference.md#FLW_TGT_HT) | Vehicle follow-me height. Note that this height is fixed *relative to the home/arming position* (not the target vehicle). Default and minimum height is 8 meters (about 26 ft)
+<a id="FLW_TGT_DST"></a>[FLW_TGT_DST](../advanced_config/parameter_reference.md#FLW_TGT_DST) | Vehicle/ground station separation in the *horizontal* (x,y) plane. Minimum allowed separation is 1 meter. Default distance is 8 meters (about 26 ft).
+<a id="FLW_TGT_FS"></a>[FLW_TGT_FS](../advanced_config/parameter_reference.md#FLW_TGT_FS) | Flight position relative to the user when follow-me mode is active.<br>- `0` = Follow from the front right.<br>- `1` = Follow from behind or trail the user (Default).<br>- `2` = Follow from the front.<br>- `3` = Follow from the front left.
+
+### Altitude Control mode explanation

```

Comment:
I would move this above the parameters and have parameters as a separate heading. It is worthy of its own topic. 
```suggestion
### Altitude Control Mode

```